cvmfs/cache.h

```diff
@@ -226,7 +226,7 @@ class CacheManager : SingleCopy {
  protected:
   CacheManager();
 
-  // Unless overwritten, Saving/Restoring states will crash the Fuse module
+  // Unless overridden, Saving/Restoring states will crash the Fuse module
```

Comment:
overwrite and override are both words, and sometimes you overwrite something to override it, but I think in this case it's just the latter.